Packs/AutoFocus/Integrations/AutofocusV2/AutofocusV2.py

```diff
@@ -1537,6 +1538,14 @@ def search_file_command(file):
     return command_results
 
 
+def get_tags_for_generic_conext(tags: dict):
+    keys = ['TagGroups', 'Aliases', 'PublicTagName', 'TagName']
+    sub_keys = ['TagGroupName']
+    generic_context_tags = {key: tags[0].get(key) for key in keys}
```

Comment:
Why tags[0]? Isn't tags of type dict?
And if it is a list, shouldn't we take the information from all list attributes and not just the first one?

Reply:
Done